<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTabWidget" name="mainTab">
      <property name="styleSheet">
       <string notr="true">font: 57 14pt &quot;Montserrat Medium&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>Pricing Details</string>
       </attribute>
       <widget class="QTableWidget" name="tablePrice">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>1061</width>
          <height>551</height>
         </rect>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 75 10pt &quot;Century Gothic&quot;;
font: 75 11pt &quot;Century Gothic&quot;;
</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::EditKeyPressed</set>
        </property>
        <column>
         <property name="text">
          <string>Site</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sosyal Medya</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ürün</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ülke</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Kalite</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tedarikçi</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Adet</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Fiyat</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Maliyet</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Maliyet Oranı</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Pricing Entries</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>10</y>
          <width>179</width>
          <height>148</height>
         </rect>
        </property>
        <property name="title">
         <string>Websites</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QComboBox" name="boxWebsite">
           <item>
            <property name="text">
             <string>IT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>IF</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pop</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>180</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Provider</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>271</width>
          <height>331</height>
         </rect>
        </property>
        <property name="title">
         <string>Details</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Social Media</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Quality</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Product</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="boxQuality">
          <item>
            <property name="text">
             <string>VIP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Real</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Bot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Custom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Easy Content</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Random</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>AI Content</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="boxProduct">
           <item>
            <property name="text">
             <string>Followers</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Likes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reel Likes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reel Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Story Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shorts Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shorts Likes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shorts Comments</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Comments</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Plays</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Playlist Likes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Monthly Listeners</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Retweets</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Shares</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Page Likes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Impressions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Profile Visits</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Poll Votes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Members</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Saves</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Impressions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Live Viewers</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last Post Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last 1 Post Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last 5 Post Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last 10 Post Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last 50 Post Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Last 100 Post Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Reactions</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Event(Thinking)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Event(Going)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Clip Views</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Watch Time</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Repin</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QComboBox" name="boxSM">
           <item>
            <property name="text">
             <string>Instagram</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tiktok</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Twitter</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Facebook</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Youtube</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Spotify</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Telegram</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Twitch</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LinkedIn</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Soundcloud</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Threads</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Discord</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vimeo</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Mixcloud</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Pinterest</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Whatsapp</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Country</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QComboBox" name="boxCountry">
           <item>
            <property name="text">
             <string>TR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Global</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>USA</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>BR</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="lineProvider">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>210</y>
          <width>201</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>250</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Service Price</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineSrvPrice">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>280</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>491</width>
          <height>601</height>
         </rect>
        </property>
        <property name="title">
         <string>Quantities</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="5">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>2500</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>5000</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>10000</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>25000</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>250</string>
           </property>
          </widget>
         </item>
         <item row="4" column="5">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>50000</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>500</string>
           </property>
          </widget>
         </item>
         <item row="5" column="5">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>100000</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>1000</string>
           </property>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>250000</string>
           </property>
          </widget>
         </item>
         <item row="7" column="5">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>500000</string>
           </property>
          </widget>
         </item>
         <item row="8" column="6">
          <widget class="QPushButton" name="btnCalculate">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(85, 170, 255);</string>
           </property>
           <property name="text">
            <string>Calculate</string>
           </property>
          </widget>
         </item>
         <item row="9" column="6">
          <widget class="QPushButton" name="btnClear">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEdit1"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEdit2"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEdit3"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEdit4"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="lineEdit5"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="lineEdit6"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineEdit7"/>
         </item>
         <item row="0" column="6">
          <widget class="QLineEdit" name="lineEdit8"/>
         </item>
         <item row="1" column="6">
          <widget class="QLineEdit" name="lineEdit9"/>
         </item>
         <item row="2" column="6">
          <widget class="QLineEdit" name="lineEdit10"/>
         </item>
         <item row="3" column="6">
          <widget class="QLineEdit" name="lineEdit11"/>
         </item>
         <item row="4" column="6">
          <widget class="QLineEdit" name="lineEdit12"/>
         </item>
         <item row="5" column="6">
          <widget class="QLineEdit" name="lineEdit13"/>
         </item>
         <item row="6" column="6">
          <widget class="QLineEdit" name="lineEdit14"/>
         </item>
         <item row="7" column="6">
          <widget class="QLineEdit" name="lineEdit15"/>
         </item>
         <item row="9" column="1" colspan="2">
          <widget class="QPushButton" name="btnMontlyCalc">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>Calculate Monthly</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1" colspan="2">
          <widget class="QPushButton" name="btnAutoCalc">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 85, 255);
color: rgb(255,255,255);</string>
           </property>
           <property name="text">
            <string>Calculate Autos</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QRadioButton" name="radioBtnTry">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>320</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TRY</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioBtnUsd">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>350</y>
          <width>121</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>USD</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>380</y>
          <width>431</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Comments</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_29">
           <property name="text">
            <string>20</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="comment1"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="comment2"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_30">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_31">
           <property name="text">
            <string>40</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="comment3"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="comment4"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>50</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_33">
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="comment5"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="comment6"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>250</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLineEdit" name="comment7"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="comment8"/>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>500</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1132</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>boxWebsite</tabstop>
  <tabstop>boxQuality</tabstop>
  <tabstop>boxSM</tabstop>
  <tabstop>boxProduct</tabstop>
  <tabstop>boxCountry</tabstop>
  <tabstop>lineProvider</tabstop>
  <tabstop>lineSrvPrice</tabstop>
  <tabstop>btnCalculate</tabstop>
  <tabstop>btnClear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
